linux_vmstat.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+#
+# igcollect - Linux virtual memory stats
+#
+# Copyright (c) 2016, InnoGames GmbH
+#
+
+from __future__ import print_function
+from socket import gethostname
+import time, sys
+
+# utility to read and parse a space delimited file (meminfo)
+def parse_split_file(filename):
+    try:
+        with open(filename, 'r') as f:
+            return [line.strip().split(None, 1) for line in f]
+    except:
```

Comment:
It is always a bad idea to `expect` anything.  You would hide things like syntax errors, or Ctrl+C, if they happen to be inside this.  I don't thing we need any reason to expect anything in here.